glue/sales_val_flagging.py

```diff
@@ -499,10 +499,6 @@ def modify_dtypes(df):
         df: df of standardized dtypes
     """
 
-    # Convert object columns to string
-    for col in df.select_dtypes("object").columns:
-        df[col] = df[col].astype("string")
-
```

Comment:
This is what was overwriting the array type conversation. I changed it and tested the an initial and manual run (writing both of the parameter dfs to athena), and after running the crawler it seems like there were no errors.

 I believe that the `Int64` conversion below is where the major error was (what this function was written for). I will double check this during a test final run but I'm fairly sure this will work.

Reply:
Sounds good! Hopefully this is enough.